Fixed mobile responsive issue

skushagra9 opened the pull request:
#925 
![Screenshot from 2024-02-23 21-42-28](https://github.com/bowtie-json-schema/bowtie/assets/120712705/32b7f9f9-8a5c-47fe-89cb-748ddf294509)
![Screenshot from 2024-02-23 21-42-33](https://github.com/bowtie-json-schema/bowtie/assets/120712705/1d850316-ddbc-4de9-9c24-fd77b6ccdf9e)

Julian commented:
Thanks. Are those screenshots before or after the change? I presume obviously after.

Can you try to briefly explain what you've done and why when submitting changes?

In particular, here are two immediate things from your screenshots:

* it looks like you made labels be vertically justified to the bottom. Probably top makes more sense for this, as they're labels, and someone shouldn't see data before they know what they're lookinga t
* What did you do to fix the table? Does it horizontally scroll, or where are the extra columns?

skushagra9 commented:
These are the pictures after the change
So , I made changes at two places 
1) commit link was overflown, image1- now its not flowing out anymore
2) and the summary table which was overflown

Both of them works perfectly fine,
and the scroll works horizontally and vertically both

Julian commented on frontend/src/components/RunInfo/RunInfoSection.tsx:
```diff
@@ -50,7 +50,7 @@ const RunInfoSection = ({ runInfo }: { runInfo: RunInfo }) => {
                       <tr key={label}>
                         <td>{label}</td>
-                        <td>
+                        <td style={{ maxWidth: "200px" }}>
```
I'm not sure pixels are the right unit to use here. Do you know that they are?

skushagra9 replied:
yes, I researched on it

skushagra9 replied:
you can try it once in the browser, by minimizing and maximinzing the window

skushagra9 replied:
okay, I understand , what I did in my latest commit was to make it more generic, otherwise pixels also work ,
Pixels are a more absolute way of defining, what I now did in my latest commit is a reletive way

skushagra9 commented:
I have added the corrected images in the description

skushagra9 commented:
@Julian , my latest commit made it more generic

Julian commented:
What is "it", what do you mean by "corrected images", corrected how?

@skushagra9 fast/sloppy work isn't what I want! Be slow, explain what you're doing clearly, and read what I'm asking you.

skushagra9 commented:
I have added an commit , which fixes the maxwidh:200px to 50vw which is more generic
By what I meant from corrected images was , that I added the images before and after solving the issue

Julian commented:
@skushagra9 I'm giving you direct feedback that you're not doing yourself any favors here. I'm not trying to be hard on you, but responding immediately with a 2 sentence message in 2 minutes right after I tell you "be slow and careful" is not a good look. Your message makes very little sense, you just said "I researched it" on whether pixels are the right unit and then immediately change your mind when I ask -- so did you or did you not research what the right answer is? If you didn't clearly, then the right thing to do is to go re-read something to learn what the right thing to do is -- and then ideally share that so I learn it too.

You also haven't addressed what I asked about vertical justification -- that's more sloppiness, since even though I should have put that somewhere distinguishable, if you were reading slower and not rushing you would have remembered there was something else to change or respond to.

I'll have a look at your work in a bit and see about merging it, but please understand clearly that sloppiness and rushing are essentially the worst things you could be doing for your chances!

skushagra9 commented:
Hey @Julian , I don't understand why did you got confused, As a token of respect I changed my approach , but there is no problem , This was a easy fix, so I made the fix quick
